genai-perf/tests/test_perf_analyzer_config.py

```diff
@@ -300,10 +296,10 @@ def test_add_perf_analyzer_args_with_non_payload_prompt_source(self):
         self._config.perf_analyzer.warmup_request_count = 30
 
         expected_args = [
-            "--stability-percentage",
-            "567",
             "--warmup-request-count",
             "30",
+            "--stability-percentage",
+            "567",
```

Comment:
My logic change here:

https://github.com/triton-inference-server/perf_analyzer/pull/366/files#diff-d1c5f6fe3374963d887eec07bbc9270be4be7ca81e1c6ccf2632c50027204eb3R249-R253

Caused these options to be appended in a different order, nothing changes otherwise.